src/sourcemap/sourcemap.zig

```diff
@@ -356,17 +356,15 @@ pub const LineColumnOffset = struct {
             std.debug.assert(i >= offset);
             std.debug.assert(i < input.len);
 
-            offset = i + 1;
+            var iter = strings.CodepointIterator.initOffset(input, i);
+            var cp = iter.nextCodepoint();
```

Comment:
you can't use Cursor? We should clean up this code, but it was done this way because there were off-by-one errors when using .next() because the offset was updated after use when usually you want the previous value

Reply:
switched back to `Cursor`. Should we change `Cursor.i` to a `usize`?